test/testcafe/spec/SelectAuthenticatorForEnroll_spec.js

```diff
@@ -228,14 +213,12 @@ test.requestHooks(mockEnrollAuthenticatorPassword)('should navigate to password
   await t.expect(enrollPasswordPage.confirmPasswordFieldExists()).eql(true);
 });
 
-test.requestHooks(requestLogger, mockEnrollAuthenticatorPassword)('select password challenge page and hit switch authenticator and re-select password', async t => {
+test.meta('v3', false).requestHooks(requestLogger, mockEnrollAuthenticatorPassword)('select password challenge page and hit switch authenticator and re-select password', async t => {
```

Comment:
Had to disable this because when switching back to authenticator list makes an API request in v3 but does not in V2. Until we update that (which is apart of this ticket https://oktainc.atlassian.net/browse/OKTA-521014) we will need to disable this test for the parity execution.